index.html

```diff
@@ -54,98 +63,110 @@ <h1 class="header intro-header">I’m Jordan Glad to see you!</h1>
-                <li>javaScript</li>                
-            </ul>
-            <a class="btn sp-button" href="#">See Project</a>
         </div>
         <div class="card card2">
             <img class="img-proj" src="images/cardsWorksSection/card2.svg" alt="Project 2 Image">
-            <h2 class="proj-header">Multi-Post Stories</h2>
-            <div class="features">
-                <div class="f f1 upper-case">canopy</div>
-                <span>.</span>
-                <div class="f f2">Back End Dev</div>
-                <span>.</span>
-                <div class="f f3">2015</div>
+            <div class="proj-content">
+                <h2 class="proj-header">Multi-Post Stories</h2>
+                <div class="features">
+                    <div class="f f1 upper-case">canopy</div>
+                    <span>.</span>
+                    <div class="f f2">Back End Dev</div>
+                    <span>.</span>
+                    <div class="f f3">2015</div>
+                </div>
+                <p class="p p-proj">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>                
+                </ul>
+                <a class="btn sp-button" href="#">See Project</a>
             </div>
-            <p class="p p-proj">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="tags">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>                
-            </ul>
-            <a class="btn sp-button" href="#">See Project</a>
         </div>
         <div class="card card3">
             <img class="img-proj" src="images/cardsWorksSection/card3.svg" alt="Project 3 Image">
-            <h2 class="proj-header">Tonic</h2>
-            <div class="features">
-                <div class="f f1 upper-case">canopy</div>
-                <span>.</span>
-                <div class="f f2">Back End Dev</div>
-                <span>.</span>
-                <div class="f f3">2015</div>
+            <div class="proj-content">
+                <h2 class="proj-header">Tonic</h2>
+                <div class="features">
+                    <div class="f f1 upper-case">canopy</div>
+                    <span>.</span>
+                    <div class="f f2">Back End Dev</div>
+                    <span>.</span>
+                    <div class="f f3">2015</div>
+                </div>
+                <p class="p p-proj">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>                
+                </ul>
+                <a class="btn sp-button" href="#">See Project</a>
```

Comment:
- [ ] [OPTIONAL] I think there is an issue with the buttons hover effect because these are showing different properties than the Figma design. I strongly suggest updating this in order to match the Figma design :smiley: :pray:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144548702-870b68fd-5148-42dd-8c1c-4d9369610ab3.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144548735-b3403136-0351-4bdf-a806-61a33b2b04ee.png)